.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,41 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/go
+{
+	"name": "Go",
+	"image": "mcr.microsoft.com/devcontainers/go:0-1.18-bullseye",
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// "features": {},
+
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Set *default* container specific settings.json values on container create.
+			"settings": {
+				"go.toolsManagement.checkForUpdates": "local",
+				"go.useLanguageServer": true,
+				"go.gopath": "/go"
+			},
+
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
+				"golang.go"
+			]
+		}
+	},
+	"features": {
+		"ghcr.io/devcontainers/features/docker-outside-of-docker:1": {},
+		"ghcr.io/devcontainers/features/git:1": {},
+		"ghcr.io/devcontainers/features/github-cli:1": {}
+	}
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	// "postCreateCommand": "go version",
+
+	// Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
+	// "remoteUser": "root"
```

Comment:
Should we remove all these comments? Or at least, let's define what `postCreateCommand` to use.

I can see we compile the project in Java: https://github.com/testcontainers/testcontainers-java/blob/dbb9be00b91f582e8fdc76e569b700c9f8fb88d0/.devcontainer/devcontainer.json#L48 and in .NET https://github.com/testcontainers/testcontainers-dotnet/blob/eb4a6d9aa267673487aa66b9951edee40c3e9006/.devcontainer/devcontainer.json#L29-L30

Reply:
I like both suggestions 👍

Reply:
Yes, let's do like this. Nowadays the format is kind of established after all.